Dockerfile.mysql

```diff
@@ -0,0 +1,4 @@
+FROM mysql:5
+
+COPY schema.sql /docker-entrypoint-initdb.d/000-schema.sql
+COPY sql/ /docker-entrypoint-initdb.d/
```

Comment:
COPY can be made into a single layer here.
i.e., `COPY schema.sql sql/ /docker-entrypoint-initdb.d/`

REF: https://docs.docker.com/engine/reference/builder/#copy

Reply:
My concern here is that the sql files in `docker-entrypoint-initdb.d/` are processed in lexicographic order, so we need to ensure `schema.sql` gets executed first. I didn't necessarily want to introduce a strangely named version of `schema.sql` into the user instructions, hence this solution.